cmd/minishift/cmd/root.go

```diff
@@ -138,6 +151,8 @@ var RootCmd = &cobra.Command{
 			log.SetOutWriter(ioutil.Discard)
 			log.SetErrWriter(ioutil.Discard)
 		}
+
+		setDefaultActiveProfile()
```

Comment:
setDefaultActiveProfile...

Isn't that just setActiveProfile() without a parameter or an explicit parameter?

Reply:
Nope, it is more than that. This is the function helps with the backward compatibility with Minishift versions which do not have profile feature.